examples/CRISP/interfold.config.yaml

```
@@ -5,26 +5,26 @@ chains:
       interfold:
-        address: "0xA51c1fc2f0D1a1b8494Ed1FE312d7C3a78Ed91C0"
-        deploy_block: 14
+        address: "0x0DCd1Bf9A1b36cE34237eEaFef220932846BCD82"
+        deploy_block: 16
       ciphernode_registry:
         address: "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707"
         deploy_block: 8
       bonding_registry:
         address: "0xa513E6E4b8f2a923D98304ec87F64353C4D5C853"
         deploy_block: 9
       bonding_registry:
```

YAMLlint (1.37.1)

[error] duplication of key "bonding_registry" in mapping

(key-duplicates)